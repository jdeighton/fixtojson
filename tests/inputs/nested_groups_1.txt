8=FIX.4.4 | 555=2 | 604=2 | 605=F7 | 605=CGYU0 | 604=2 | 605=F7 | 605=CGYM0 | 10=209