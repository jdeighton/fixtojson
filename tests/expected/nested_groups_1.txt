{"8":"FIX.4.4","555":[{"604":[{"605":"F7"},{"605":"CGYU0"}]},{"604":[{"605":"F7"},{"605":"CGYM0"}]}],"10":"209"}
